res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -381,6 +416,21 @@ PioneerDDJSX.init = function(id) {
             PioneerDDJSX.initDeck(index);
         }
     }
+
+    // init effects section:
+    PioneerDDJSX.effectUnit = [];
+    PioneerDDJSX.effectUnit[1] = new PioneerDDJSX.EffectUnit(1);
```

Comment:
Do you still have the same issue assigning `PioneerDDJSX.effectUnit[1].dryWetKnob.input` after changing this?